src/pages/create.tsx

```diff
@@ -114,14 +118,20 @@ const SignUpPage = () => {
           <Button
             onClick={() => {
               if (displayError) {
-                // error, check identifier, bad file?
+                // TODO: error, check identifier, bad file?
               } else {
-                const data = {
+                const programInput: CreateProgramInput = {
                   name: programName,
+                  description: "", // a lotta dummy strings for now since the form doesn't specify them
                   slug: programIdentifier,
                   iconUrl: programLogo,
+                  homepage: programIdentifier,
+                  mentorProfileSchemaJson: "",
+                  menteeProfileSchemaJson: "",
+                  mentorApplicationSchemaJson: "",
+                  public: true,
                 };
-                // useCreateProgram(data); TODO: actually plug this into backend
+                createProgram({ variables: { data: programInput } });
                 setStage(stage + 1);
```

Comment:
Generally its good practice to do setStage((prev) => prev + 1); because the increment doesnt happen immediately on call and stage might change during the time